InflectorKit/InflectorKit.h

```diff
@@ -0,0 +1,7 @@
+#import <UIKit/UIKit.h>
```

Comment:
I just realized this should be Foundation. That's what I get for copying and pasting. I'll fix it soon.